lesson16/main.js

```diff
@@ -128,27 +134,61 @@ function createNewIconContainer({ date }) {
   return newIconContainer;
 }
 
-function createCommentIconContainer({ comment }) {
-  const comments = comment;
+function createCommentIconContainer({ id, comment }) {
   const commentIconContainer = document.createElement("div");
-  commentIconContainer.classList.add("comment-container");
-  if (comments.length > 0) {
+  commentIconContainer.classList.add("comment-icon-container");
+  if (comment.length > 0) {
     const commentIcon = document.createElement("img");
     commentIcon.classList.add("comment-icon");
     commentIcon.src = "./img/comment.png";
     commentIcon.width = "14";
     commentIcon.height = "14";
+    commentIcon.dataset.id = id;
     commentIconContainer.appendChild(commentIcon);
 
     const numOfComments = document.createElement("div");
     numOfComments.classList.add("comment-num");
-    numOfComments.textContent = comments.length;
+    numOfComments.textContent = comment.length;
     numOfComments.alt = "コメント数";
     commentIconContainer.appendChild(numOfComments);
   }
   return commentIconContainer;
 }
 
+function createComments({ article }) {
+  article.forEach(({ comment, id }) => {
+    const anArticleCommentsContainer = document.createElement("div");
+    anArticleCommentsContainer.id = id;
+    anArticleCommentsContainer.classList.add("comment-container");
+    // anArticleCommentsContainer.style.display = "none";
+    comment.forEach((comment) => {
+      const aComment = createComment(comment);
+      anArticleCommentsContainer.appendChild(aComment);

```

Comment:
Q. 
命名に冠詞を使うのはあまり見たことがない(私の経験上🙇🏻‍♀️)のですが...何か理由はありますか？👀
```suggestion
      articleCommentsContainer.appendChild(aComment);
```

Reply:
Thank you sae-san :)
I changed that "const comment = createComment(comments);"
I added that for my own readability.
I removed a or an from my code.
556b36b5895a9f04ced1f7c3e23cea6aca932bce